spec/unit/pops/loaders/loader_paths_spec.rb

```diff
@@ -38,6 +38,7 @@ def foo; end;
     expect(effective_paths.size).to eq(1)
     expect(module_loader.path_index.size).to eq(1)
     path_index = module_loader.path_index
-    expect(path_index).to include(File.join(module_dir, 'lib', 'puppet', 'functions', 'foo4x.rb'))
+    expected = File.join(module_dir, 'lib', 'puppet', 'functions', 'foo4x.rb')
+    expect(path_index).to include(Dir.glob(expected).first)
```

Comment:
The use of Dir.glob exposes an implementation detail. Shouldn't this go via `FileSystem::path_string(expected)` instead? (Or did I misunderstand).